src/me/EtienneDx/RealEstate/Transactions/TransactionsStore.java

```diff
@@ -122,7 +124,12 @@ public void saveData()
             config.set("Lease." + cl.claimId, cl);
         try
         {
-			config.save(new File(this.dataFilePath));
+			File cfgFile = new File(this.dataFilePath);
+			if(!cfgFile.exists())
+			{
+				com.google.common.io.Files.touch(cfgFile);
+			}
+			config.save(cfgFile);

```

Comment:
I don't think this change is required, since the YamlConfiguration creates the file if it doesn't exist